### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by knatsakis**
- Replace @knatsakis with @netdata/automation 
- Remove @cakrit

### CODE REVIEW

**Comment by cakrit** [comment]
Can u please also replace @VLegakis with @jacekkolasa?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,58 +2,58 @@
...
+.lgtm.yml @cosmix @netdata/automation
+.eslintrc @cosmix @netdata/automation
+.eslintignore @cosmix @netdata/automation
+.csslintrc @cosmix @netdata/automation
+.codeclimate.yml @cosmix @netdata/automation
+.codacy.yml @cosmix @netdata/automation
+netdata.spec.in @cosmix @netdata/automation
+netdata-installer.sh @cosmix @netdata/automation
+netlify.toml @cosmix
 package.json @VLegakis
```
Comment (jacekkolasa) [inline .github/CODEOWNERS]:
```suggestion
package.json @jacekkolasa
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,58 +2,58 @@
...
+docs/ @cosmix @joelhans
+health/ @thiagoftsm @vlvkobal @cosmix
+health/health.d/ @thiagoftsm @vlvkobal @cosmix
+health/notifications/ @Ferroin @thiagoftsm @cosmix
+libnetdata/ @thiagofsm @mfundul @cosmix
+packaging/ @cosmix @netdata/automation
+registry/ @VLegakis @cosmix
+streaming/ @thiagoftsm @vlvkobal @cosmix
+web/ @thiagoftsm @mfundul @vlvkobal @cosmix
+web/gui/ @VLegakis @cosmix
```
Comment (jacekkolasa) [inline .github/CODEOWNERS]:
```suggestion
web/gui/ @jacekkolasa @cosmix
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -2,58 +2,58 @@
...
+collectors/cups.plugin/ @simonnagl @vlvkobal @thiagoftsm @cosmix
+daemon/ @thiagoftsm @mfundul @cosmix
+database/ @mfundul @thiagoftsm @cosmix
+docs/ @cosmix @joelhans
+health/ @thiagoftsm @vlvkobal @cosmix
+health/health.d/ @thiagoftsm @vlvkobal @cosmix
+health/notifications/ @Ferroin @thiagoftsm @cosmix
+libnetdata/ @thiagofsm @mfundul @cosmix
+packaging/ @cosmix @netdata/automation
+registry/ @VLegakis @cosmix
```
Comment (jacekkolasa) [inline .github/CODEOWNERS]:
```suggestion
registry/ @jacekkolasa @cosmix
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -2,58 +2,58 @@
...
-collectors/node.d.plugin/ @VLegakis @cakrit @cosmix
-collectors/node.d.plugin/fronius/ @ccremer @cakrit @cosmix
-collectors/node.d.plugin/snmp/ @VLegakis @cakrit @cosmix
-collectors/node.d.plugin/stiebeleltron/ @ccremer @cakrit @cosmix
+build/ @cosmix @netdata/automation
+collectors/ @vlvkobal @mfundul @cosmix
+collectors/charts.d.plugin/ @cosmix
+collectors/freebsd.plugin/ @vlvkobal @thiagoftsm @cosmix
+collectors/macos.plugin/ @vlvkobal @thiagoftsm @cosmix
+collectors/node.d.plugin/ @VLegakis @cosmix
```
Comment (jacekkolasa) [inline .github/CODEOWNERS]:
```suggestion
collectors/node.d.plugin/ @jacekkolasa @cosmix
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -2,58 +2,58 @@
...
+collectors/node.d.plugin/fronius/ @ccremer @cosmix
+collectors/node.d.plugin/snmp/ @VLegakis @cosmix
```
Comment (jacekkolasa) [inline .github/CODEOWNERS]:
```suggestion
collectors/node.d.plugin/snmp/ @jacekkolasa @cosmix
```

**Pull request merged**